examples/addsvc/client/grpc/client.go

```diff
@@ -4,19 +4,25 @@ import (
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 
-	"github.com/go-kit/kit/examples/addsvc/pb"
+	"github.com/go-kit/kit/endpoint"
 	"github.com/go-kit/kit/examples/addsvc/server"
 	"github.com/go-kit/kit/log"
+	grpctransport "github.com/go-kit/kit/transport/grpc"
 )
 
 // New returns an AddService that's backed by the provided ClientConn.
 func New(ctx context.Context, cc *grpc.ClientConn, logger log.Logger) server.AddService {
-	return client{ctx, pb.NewAddClient(cc), logger}
+	var sumReply, concatReply string
+	sumClient := grpctransport.NewClient(cc, "Add", "sum", encodeSumRequest, decodeSumResponse, sumReply)
+	concatClient := grpctransport.NewClient(cc, "Add", "concat", encodeConcatRequest, decodeConcatResponse, concatReply)
+
+	return client{ctx, sumClient.Endpoint(), concatClient.Endpoint(), logger}
```

Comment:
I would follow the same coding style like httpjson/client.go and do something like this:
```
return client{
    Context: ctx,
    Logger:  logger,
    sum: grpctransport.NewClient(
        ...,
    ).Endpoint(),
    concat: grpctransport.NewClient(
        ...,
    ).Endpoint(),
}